src/component/common/Table/TableSearch/TableSearch.styles.ts

```diff
@@ -0,0 +1,59 @@
+import { makeStyles } from 'tss-react/mui';
+
+export const useStyles = makeStyles()(theme => ({
+    fieldWidth: {
```

Comment:
We might want to rename these classes to something more appropriate. Maybe `searchField`, `searchFieldExpanded` and `searchFieldCollapsed`?